### PR TITLE
HCX-58 Add PDF conversion option to Imprint print interface

### PR DESCRIPTION
**Pull request by TimAle**

### CODE REVIEW

File: src/org/camunda/latera/bss/connectors/Imprint.groovy [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Imprint {
     )
   }
 
-  def print(CharSequence template, Map data = [:]) {
+  def print(CharSequence template, Map data = [:], convertToPDF = false) {
```
Comment (dolfinus) [inline src/org/camunda/latera/bss/connectors/Imprint.groovy]: Please add Boolean type here

Comment (TimAle) [reply src/org/camunda/latera/bss/connectors/Imprint.groovy]: Sure thanks

File: src/org/camunda/latera/bss/connectors/Imprint.groovy [hunk 2]
```diff
@@ -36,7 +36,7 @@ class Imprint {
     )
   }
 
-  def print(CharSequence template, Map data = [:]) {
+  def print(CharSequence template, Map data = [:], Boolean convertToPDF = false) {
     logger.info('Printing begin...')
```
Comment (sugarfree1) [inline src/org/camunda/latera/bss/connectors/Imprint.groovy]: i wonder how it looks when you have thousands of requests.. :)

File: src/org/camunda/latera/bss/connectors/Imprint.groovy [hunk 3]
```diff
@@ -54,7 +54,8 @@ class Imprint {
                   monthFull     : format(now, MONTH_FULL_FORMAT, this.locale),
                   todayFull     : format(now, FULL_DATE_FORMAT,  this.locale),
                   homsOrderCode : execution.getVariable('homsOrderCode')
-                ] + data
+                ] + data,
+      options   : [ convert_to_pdf : convertToPDF ]
```
Comment (sugarfree1) [inline src/org/camunda/latera/bss/connectors/Imprint.groovy]: alignment

File: src/org/camunda/latera/bss/connectors/Imprint.groovy (hunk 2 above)
Comment (dolfinus) [reply src/org/camunda/latera/bss/connectors/Imprint.groovy]: What do you mean?

**Pull request merged**